common/src/main/kotlin/entity/DiscordGuild.kt

```diff
@@ -390,16 +397,16 @@ public enum class SystemChannelFlag(public val code: Int) {
     /** Suppress server boost notifications. **/
     SuppressPremiumSubscriptions(1.shl(1)),
 
-    /** Suppress guild reminder notifications **/
+    /** Suppress server setup tips. **/
```

Comment:
```suggestion
    /** Hide server setup tips. **/
```

Reply:
i just copy pasted this: https://discord.com/developers/docs/resources/guild#guild-object-system-channel-flags

Reply:
That is discord Dev docs being annoying because that's not consistent